scipy/_lib/_numpy_compat.py

```diff
@@ -88,6 +88,41 @@ def broadcast_to(array, shape, subok=False):
         return _broadcast_to(array, shape, subok=subok, readonly=True)
 
 
+if NumpyVersion(np.__version__) >= '1.11.0':
+    def get_randint(random_state):
+        return random_state.randint
+else:
+    # In NumPy versions previous to 1.11.0 the randint funtion and the randint
+    # method of RandomState does only work with int32 values.
+    def get_randint(random_state):
+        def randint_patched(args, kwargs):
```

Comment:
This should be `*args, **kwargs` here, right?

Reply:
Yes, of course. I changed it.